tensorflow/lite/kernels/fully_connected.cc

```diff
@@ -78,7 +78,8 @@ inline TfLiteStatus CheckTypes(TfLiteContext* context,
 
   // optional bias tensor.
   const bool is_optional_bias_float = !bias || (bias->type == kTfLiteFloat32);
-  const bool is_optional_bias_int = !bias || (bias->type == kTfLiteInt32);
+  const bool is_optional_bias_int =
+      !bias || (bias->type == kTfLiteInt32) || (bias->type == kTfLiteInt64);
```

Comment:
curious why bias needs to be int64?

Reply:
To avoid overflow we need 40-bit accumulator, because we have 16-bit activations and 8-bit weights, so we use int64 as an accumulator and, hence, we use bias as int64 not to lose the precision.